### PR TITLE
Add song list history & songs folder consistency fix

### PR DESCRIPTION
**Pull request by Cactooz**
- Added the history of the songs to the end of the README file.
- Fixed so the songs folder is always written like /songs in the README file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,6 +19,7 @@ You can also find information on how to contribute to this repository, either by
   - [Song scripts](#song-scripts)
   - [Project scripts](#project-scripts)
 - [Getting help](#getting-help)
+- [History](#history)
```
Comment (StelFoog) [inline README.md:22]: Would prefer it to be `Background` instead of `History`

**Review by StelFoog — COMMENTED** [review]
Mostly looks good

File: README.md (hunk 1 above)
Comment (Cactooz) [reply README.md]: Resolved with fixup commit.

**Review by StelFoog — APPROVED** [review]
LGTM

**Pull request merged**